db/dbProvider.js

```diff
@@ -0,0 +1,19 @@
+class DBProvider {
+    constructor({ client }) {
+        this.client = client;
+    }
+
+    async insert(data) {
+        const db = this.client.db(process.env.MONGO_DB);
```

Comment:
Pass envs through constructor

Reply:
btw u can create DBProvider with already selected collection
Then u can create provider duplicates when needed to use another collection